content/en/docs/code/cri-api-version-skew-policy.md

```diff
@@ -0,0 +1,123 @@
+---
+content_type: "reference"
+title: CRI API version skew policy
+weight: 51
+---
+
+CRI is a plugin interface which enables the kubelet to use a wide variety of container runtimes,
```

Comment:
this file is mostly copied from https://github.com/kubernetes/kubernetes/pull/129789

Reply:
so it was approved for a long time, just moving it in the proper place